assembl/models/generic.py

```diff
@@ -454,13 +454,16 @@ def get_body_as_text(self):
             log.error("What is this mimetype?" + mimetype)
             return body
 
-    def maybe_translate(self, pref_collection):
+    def maybe_translate(self, pref_collection=None):
```

Comment:
Hmm, maybe this is a way to do it. But, I would expect that `pref_collection` to always be passed. In fact, under every request, no matter the caller, there is a `request.lang_prefs`. You can use the `setCurrentFromLocale` with the `has_lang` locale. And then pass this collection that is on the request to `maybe_translate`. It's a bit spaghetti, I know. But I feel that if you need to force a lang_collection, you can force it for the request, and of course, pass it to the translate.

Reply:
Look in `assembl/models/auth.py` for `LanguagePreferenceCollection`.

Reply:
This was actually my suggestion; without the pref_collection, we get the discussion's table, which translates to all discussion languages (which is what he needs for the report.)

Reply:
Hmm, alright fair. So, not biasing against a userpref, but instead all supported languages of a debate.

Reply:
@maparent @ayyazdaniaryan I need only one language in fact (that is chosen by the user that requests the export so I'd prefer to translate only to this locale (like I said in my first comment: https://github.com/assembl/assembl/pull/574#issue-226614165). What would be the best way to do it in your opinion?

Reply:
Suggestion: current (admin) user presumably has a target language. Use his prefs.